x-pack/test/security_solution_api_integration/test_suites/detections_response/utils/retry_delete_by_query_conflicts.ts

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { DeleteByQueryResponse } from '@elastic/elasticsearch/lib/api/types';
+import { ToolingLog } from '@kbn/tooling-log';
+
+// Number of times to retry when conflicts occur
+export const RetryForConflictsAttempts = 2;
+
+// Delay between retries when conflicts occur
+export const RetryForConflictsDelay = 200;
```

Comment:
Nit: Normally we name constant literals `THIS_WAY`.

Reply:
Yes, thanks, renaming.